Makefile

```diff
@@ -30,3 +30,10 @@ pi-dangerous.img: ubuntu-core-desktop-22-pi-dangerous.model $(EXTRA_SNAPS)
 	tar czSf $@ $<
 
 .PHONY: all
+
+clean:
+	sudo rm -rf img
+	sudo rm -rf image2
```

Comment:
I don't seem to have image2 or output on my system - is that something you did locally?

Reply:
It's part of the "installer_builder" branch...

Reply:
Removed.